mage/mage/be_dev.clj

```diff
@@ -1,7 +1,7 @@
 (ns mage.be-dev
   (:require
    [bencode.core :as bencode]
-   #_:clj-kondo/ignore
+   ^:clj-kondo/ignore
```

Comment:
I had to change this because of a new pre-commit hook I merged in from master. Not really my change. As long as the kondo is still appeased, this is not a problem.

Reply:
👋 Whoops, that ones on me!